### DISCUSSION TITLE
SUGGEST HIGH GAIN MICROPHONE

### DISCUSSION THREAD

**Discussion by aqildad-create** [pschatzmann/arduino-audio-tools#874] [General]
i want to pick a sound from around 2 feet  .

 i use INMP441 and volume.setVolume(60 )  then its working but it add lot noise due to high volume increase . 

so please suggest a microphone which have a pick up of around 2 feet.

**Comment by Doekle** [comment]
There are no microphones with certain pick up distances, only sensitivity is a parameter. 
I guess that you want to record a soft sound? the INMP441 mics are quite low noise inherently so this should be fine. Can you check if normal speech/sounds are recorded well? Maybe a wrong I2S setting, or you used the wrong channel (and did you solder the W/S signal to the specified channel?) . For very low noise, you can add a small capacitor across the power supply pins of the microphone (100nF is already ok for removing chirping sounds)

**Reply by aqildad-create** [reply]
thanks for msg and reply . 

i google soft sound but i am still confuse about soft sound.

i am recording the  normal speech . with near to microphone  the recorded is very good and no noise . 

But  with around 2 feet from microphone the speaker volume   is very low so i have to set the volume to around 60 but when when i set to 60 i can clearly listening high noise with speech . 

i need a sound pick from 1 to 2 feet .  

i use default i/o pins use in the API.
PIN_I2S_BCK = 14
PIN_I2S_WS = 15
PIN_I2S_DATA_IN = 32

You can see from the board schematic on board 100n capacitor is already place . 


![image](https://github.com/pschatzmann/arduino-audio-tools/assets/65456998/c0e090da-109f-409a-a309-fc2b76325d62)


![image](https://github.com/pschatzmann/arduino-audio-tools/assets/65456998/5f7b01bf-a7f9-412d-9501-0f75950c4d64)
